src/object/cli_obj.c

```diff
@@ -1962,8 +1962,8 @@ obj_ec_recov_cb(tse_task_t *task, struct dc_object *obj,
 		 * singv recovery without fetch from server ahead - when
 		 * some targets un-available.
 		 */
```

Comment:
above comment should remove, I'll remove it at a following PR or when refresh this PR.